nibabel/streamlines/__init__.py

```diff
@@ -0,0 +1,131 @@
+import os
+from ..externals.six import string_types
+
+from .header import Field
+from .compact_list import CompactList
+from .tractogram import Tractogram, LazyTractogram
+
+from .trk import TrkFile
+#from .tck import TckFile
+#from .vtk import VtkFile
+
+# List of all supported formats
+FORMATS = {".trk": TrkFile,
+           #".tck": TckFile,
+           #".vtk": VtkFile,
+           }
+
+
+def is_supported(fileobj):
+    ''' Checks if the file-like object if supported by NiBabel.
+
+    Parameters
+    ----------
+    fileobj : string or file-like object
+        If string, a filename; otherwise an open file-like object pointing
+        to a streamlines file (and ready to read from the beginning of the
+        header)
+
+    Returns
+    -------
+    is_supported : boolean
+    '''
+    return detect_format(fileobj) is not None
+
+
+def detect_format(fileobj):
+    ''' Returns the StreamlinesFile object guessed from the file-like object.
+
+    Parameters
+    ----------
+    fileobj : string or file-like object
+        If string, a filename; otherwise an open file-like object pointing
+        to a tractogram file (and ready to read from the beginning of the
+        header)
+
+    Returns
+    -------
+    tractogram_file : ``TractogramFile`` class
+        Returns an instance of a `TractogramFile` class containing data and
+        metadata of the tractogram contained from `fileobj`.
+    '''
+    for format in FORMATS.values():
+        try:
+            if format.is_correct_format(fileobj):
+                return format
+
+        except IOError:
+            pass
+
+    if isinstance(fileobj, string_types):
+        _, ext = os.path.splitext(fileobj)
+        return FORMATS.get(ext.lower())
+
+    return None
+
+
+def load(fileobj, lazy_load=False, ref=None):
+    ''' Loads streamlines from a file-like object in voxel space.
```

Comment:
Sentence a bit hard to read - maybe `Loads streamlines in voxel space from a file-like object`.